src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import { Todo as TodoType } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+interface Props {
+  todo: TodoType;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article className={
+      todo.completed === true
+        ? 'TodoInfo TodoInfo--completed'
+        : 'TodoInfo'
+    }
```

Comment:
```suggestion
    <article className={
      todo.completed
        ? 'TodoInfo TodoInfo--completed'
        : 'TodoInfo'
    }
```
Имхо можно без === true, так как там и так булев тип приходит. Также можно подумать над использованием либы classNames

Reply:
foxed